lib/wpt-runner.js

```diff
@@ -208,3 +218,43 @@ function recursiveReaddir(dirPath) {
     });
   });
 }
+
+function getDummyCodeOfTestDriver() {
+  return `
```

Comment:
Perhaps it'd be better to move this to its own file, and fs.readFileSync it at the top of wpt-runner.js.